src/index.html

```diff
@@ -1,13 +1,352 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:ital,wght@0,400;0,500;0,700;1,300&display=swap"
+          rel="stylesheet">
+    <title>THE MET</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="shortcut icon" href="./images/icons/faviconV2.png"
+          type="image/png">
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img src="./images/logoMet.svg"
+                   alt="met logo"
+                   class="top-bar__logo"
+              >
```

Comment:
Need to fix code style everywhere 
```suggestion
              <img
                src="./images/logoMet.svg"
                alt="THE MET logo"
                class="top-bar__logo"
              >
```